api/_globals/fetch.json

```diff
@@ -384,6 +384,42 @@
             "deprecated": false
           }
         }
+      },
+      "stream_body_support": {
```

Comment:
Possible duplicate of https://github.com/mdn/browser-compat-data/blob/538fabd90821e0d10e5c65fa9bbecf004aaec233/api/Request.json#L362-L401.